R/lifecycle.R

```diff
@@ -22,6 +17,8 @@ use_lifecycle <- function() {
       Turn on roxygen2 markdown support {ui_code('use_roxygen_md()')}")
   }
 
+  use_package("lifecycle")
```

Comment:
I decided to add a hard dependency, since you'll need that if you want to use `deprecate_warn()` etc, and lifecycle is very lightweight.